errors/errors_task.go

```diff
@@ -80,6 +80,19 @@ func (err *TaskNameConflictError) Code() int {
 	return CodeTaskNameConflict
 }
 
+type TaskNameFlattenConflictError struct {
+	TaskName string
+	Include  string
+}
+
+func (err *TaskNameFlattenConflictError) Error() string {
+	return fmt.Sprintf(`task: Found multiple tasks (%s) included by "%s""`, err.TaskName, err.Include)
```

Comment:
Do you think about `CodeTaskNameConflict` or another one ?
At first I used `CodeTaskNameConflict` but I found more readable to distinct both. That being said, it would make sense, so I'll give a try to adjust the message